audio-worklet/basic/event-processor-state.html

```diff
@@ -0,0 +1,71 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>
+      Event and AudioWorkletProcessorState | Chrome WebAudio Samples
+    </title>
+    <link rel="stylesheet" href="../../resources/base.css">
+    <script src="../lib/audio-worklet-helper.js"></script>
+  </head>
+  <body>
+    <a href="../index.html" class="link-small">« AudioWorklet home</a>
+    <section>
+      <h1>
+        Event and AudioWorkletProcessorState
+      </h1>
+      <p>
+        A simple demonstration of <a href=
+        "https://webaudio.github.io/web-audio-api/#idl-def-audioworkletprocessorstate">
+        AudioWorkletProcessorState</a> and how to catch <code>
+        processorstatechange</code> event.
+      </p>
+      <p>
+        <strong>NOTE</strong>: Open up the console to see the events being
+        fired.
```

Comment:
A brief description on what is expected in the console would be really helpful to indicate if things are working.

Reply:
Done.

Reply:
Done.